skops/io/_general.py

```diff
@@ -212,7 +216,7 @@ def _get_function_name(self) -> str:
         )
 
     def get_unsafe_set(self) -> set[str]:
-        if self.trusted is True:
+        if self.trusted is True or self._get_function_name() in self.trusted:
```

Comment:
Just my personal preference, but operator precedence can sometimes be tricky.

```suggestion
        if (self.trusted is True) or (self._get_function_name() in self.trusted):
```